src/main/java/com/mojang/brigadier/StringReader.java

```diff
@@ -239,4 +239,21 @@ public void expect(final char c) throws CommandSyntaxException {
         }
         skip();
     }
+
+    public void expect(final String s) throws CommandSyntaxException {
+        if(!canRead(s.length()) || !getRemaining().startsWith(s)) {
+            throw CommandSyntaxException.BUILT_IN_EXCEPTIONS.readerExpectedSymbol().createWithContext(this, s);
+        }
+        this.cursor += s.length();
+    }
+
+    public String expectOption(final String ...opts) throws CommandSyntaxException {
+        for(String s: opts) {
```

Comment:
Style issue: do `for (String s: opts) {`

(I'm sorry - this is my fault for not commiting a style guide yet. I'll do that as soon as I can.)